features/addressing/managing_zones/removing_and_adding_zone_members_that_have_the_same_code_in_more_than_one_zone.feature

```diff
@@ -0,0 +1,26 @@
+@managing_zones
+Feature: Removing and adding zone members that have the same code in more than one zone
```

Comment:
For me, this should be only an API test, because the problem is that we set the identifier to the code field. The test should verify that we can get a zone member if there are two zones with the same country (zone member). WDYT?

Reply:
I'm also not sure that this scenario fully covers the problem we are trying to fix. Or at least it could be simplified to only adding the zone member with the same code that exists for other member in other zone, this removing part seems to be not necessary.

Reply:
yeah I changed the context as it should foxus more on the `re-adding` as the problem occurs in that case